### ISSUE TITLE
[BUG] - wrong question description and wrong inputs of test cases

### ISSUE THREAD

**Issue by zwei2016** [LeetCode-Feedback/LeetCode-Feedback#22142]
### LeetCode Username

user5091

### Problem Number, Title, and Link

1506. Find Root of N-Ary Tree: https://leetcode.com/problems/find-root-of-n-ary-tree/description/?envType=study-plan-v2&envId=premium-algo-100

### Bug Category

Problem description

### Bug Description

This problem is wrongly described. My understanding is that all the nodes of a tree is given, and I should reconstruct this tree and return its root, 
In the example 1, the input is : tree = [1,null,3,2,4,null,5,6], 
But when I print the input through coding, I get this:

> [<__main__.Node object at 0x7f430df56890>, <__main__.Node object at 0x7f430df56810>, <__main__.Node object at 0x7f430df56790>, <__main__.Node object at 0x7f430df56910>, <__main__.Node object at 0x7f430df567d0>, <__main__.Node object at 0x7f430df56750>]

It is very clear that the input is not as described. In the description, the input contains six nodes and two 'null'. But in the true input for coding, the two 'null' is missing. 

In summary, this question has a misleading description and wrong inputs for tests (either or / both). Please verify, thank you.
 

### Language Used for Code

Python/Python3

### Code used for Submit/Run operation

```bash
class Solution:
    def findRoot(self, tree: List['Node']) -> 'Node':
        print(tree)
        head = tree[0]
        queue = deque()
        cur = None
        for node in tree:
            if node:
                if cur:
                    cur.children.append(node)
                queue.append(node)
            elif not node:
                cur = queue.popleft()

        return head
```

### Expected behavior

1. re-construct the tree from a list of nodes
2. inputs of testcases should contain 'null' as claimed
